packages/patrol_cli/lib/src/android/android_test_backend.dart

```diff
@@ -128,13 +128,14 @@ class AndroidTestBackend {
         (line) async {
```

Comment:
shouldn't we pass the entire flutter command here and invoke it along with its arguments to avoid the same problem that was in #2320?

Reply:
I also see we do the same thing in `buildApkConfigOnly`